frontend/src/i18n/en.json

```diff
@@ -1,15 +1,74 @@
 {
   "login": {
-    "buttontext": "Sign In",
+    "buttontext": "Login",
     "connectionProblems": "Connection problems",
-    "email": "Email",
+    "email": "E-mail",
     "invalidData": "Wrong Email or Password",
     "invalidPassword": "The password is min. 8 characters",
     "invalidemail": "Invalid email",
     "noEmail": "Email is required",
     "password": "Password",
-    "title": "Sign in"
+    "title": "Login"
   },
-  "registration": "registration",
-  "text": "text"
+
+  "registration": {
+    "title": "Registration",
+    "fullName": "Name",
+    "privatePerson": "Private person",
+    "legalPerson": "Legal person",
+    "taxNumber": "Tax number",
+    "password": "Password",
+    "passwordConfirmation": "Password again",
+    "email": "E-mail",
+    "buttontext": "Registration",
+    "alreadyRegistered": "Already registered?",
+    "agreement": "I accept the",
+    "linkAszf": " GTC",
+    "and": " and the ",
+    "linkadat": " conditions.",
+    "address": "Address/Company seat",
+    "invalidemail": "Email address is not valid",
+    "login": "Login"
+  },
+  "companyAdmin": {
+    "newProduct": "New product",
+    "edit": "Edit",
+    "remove": "Remove",
+    "search": "Search"
+  },
+  "superAdmin": {
+    "createNewUser": "Create a new user",
+    "name": "Name",
+    "active": "Active",
+    "inactive": "Inactive",
+    "dateOfRegistration": "Date of registration",
+    "editBasicData": "Edit basic data",
+    "editProducts": "Edit products",
+    "delete": "Delete",
+    "confirmDelete": "Confirm delete?",
+    "search": "Search"
+  },
+  "datasheet": {
+    "descripotion": "Description",
+    "Events": "Events",
```

Comment:
Legyünk következetesek, ez is legyen kisbetűs.

Reply:
kijavítottam

Reply:
Ez megint nagybetűs.